.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# CODEOWNERS Best Practices
+# 1. Per Github docs: "Order is important; the last matching pattern takes the most precedence."
+# Please define less specific codeowner paths before more specific codeowner paths in order for the more specific rule to have priority
+
+# Root
+* @smartcontractkit/research @kaleofduty
+
+# Tooling
```

Comment:
Actually should the `go.mod` and `go.sum` files be updateable by anyone? If not, we're kinda back to square one atleast for the beginning till this library matures needing research merging every PR

Reply:
added